app/helpers/patient_helper.rb

```diff
@@ -28,6 +32,29 @@ def normalize_and_get_state_name(name)
     state_names[normalize_name(name)] || nil
   end
 
+  # This function will attempt to match the input to a language in the system
+  # PARAM: `lang` can be a three-letter iso-639-2t code, a two-letter iso-639-1 code, or the name (not case sensitive)
+  # PARAM EXAMPLES: 'eng', 'en', 'English', 'ENGLISH' <-- All will map to 'eng'
+  # RETURN VALUE: `nil` if unmatchable, else the three-letter iso code ('eng')
+  def normalize_and_get_language_name(lang)
```

Comment:
I think you are going to want to use this function in the `import_verifier.rb` file similar to what we do with `address_state` [here](https://github.com/SaraAlert/SaraAlert/blob/82f3661e2d5f9c3c2a57cf42897c410e1da3f0d5/test/system/roles/public_health/dashboard/import_verifier.rb#L134). I think some of the system tests are failing because we are comparing the original input language (something like "Spanish") to the value saved ("spa").

Reply:
Fixed.